src/index.html

```diff
@@ -17,16 +17,28 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
```

Comment:
Add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading.